RELEASE.md

```diff
@@ -73,24 +73,18 @@ Highlights:
     $ git checkout -b <branch>
     ```
 
-1. Remove the `-dev` suffix from the version number found in the `VERSION` file. *Note*: there must not be a space after the `-i`.
-
-    ```
-    $ sed -i'' -e 's/-dev//' VERSION
-    ```
-
-1. Commit `VERSION` changes and push your branch
-1. Issue a new PR with the `Expeditor: Exclude from Changelog` label and await approval (in the form of a [dank gif](http://imgur.com/X0sNq)) from two maintainers
-1. Pull master once again once the PR is merged into master
-1. Create & push a Git tag
+1. Remove the `-dev` suffix from the version number found in the `VERSION` file and save.
+1. Commit the `VERSION` change and push your branch.
+1. Issue a new PR with the `Expeditor: Exclude from Changelog` label and await approval (in the form of a [dank gif](http://imgur.com/X0sNq)) from two maintainers.
+1. Pull master once again once the PR is merged into master.
+1. Create & push a Git tag:
```

Comment:
Addition of missing periods and colons:
![tenor-5456706](https://user-images.githubusercontent.com/207178/44916006-8bb16a00-ad02-11e8-9e00-215260cabf7d.gif)